src/app/components/Image/index.tsx

```diff
@@ -66,17 +65,20 @@ const Image = ({
   const [isLoaded, setIsLoaded] = useState(false);
   const showPlaceholder = placeholder && !isLoaded;
   const hasDimensions = width && height;
-  const [aspectRatioX, aspectRatioY] =
-    (aspectRatio && aspectRatio) ||
-    (hasDimensions && [width, height]) ||
-    DEFAULT_ASPECT_RATIO;
+  const hasFixedAspectRatio = !!aspectRatio || !!hasDimensions;
```

Comment:
Nice, I like this variable name, and the corresponding style names - makes it a lot clearer and easier to understand what's going on 👍